### PR TITLE
[NTTA] Remove implementation exit

### PR DESCRIPTION
**Pull request by thomas-kaltenbach**
Field "implementation exit" is not needed anymore. 
Created new aff version of ntta.

### CODE REVIEW

File: file-formats/ntta/README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 File | Cardinality | Definition | Schema | Example
 :--- | :---  | :--- | :--- | :---
 `<name>.ntta.json` | 1 | [`zif_aff_ntta_v1.intf.abap`](./type/zif_aff_ntta_v1.intf.abap) | [`ntta-v1.json`](./ntta-v1.json) | [`z_aff_example_ntta.ntta.json`](./examples/z_aff_example_ntta.ntta.json)
+`<name>.ntta.json` | 1 | [`zif_aff_ntta_v2.intf.abap`](./type/zif_aff_ntta_v1.intf.abap) | [`ntta-v2.json`](./ntta-v1.json) | [`z_aff_example_ntta.ntta.json`](./examples/z_aff_example_ntta.ntta.json)
```
Comment (huber-nicolas) [inline file-formats/ntta/README.md:6]: Can you please do 2 tables in the readme as it is done for [ SITO](https://github.com/SAP/abap-file-formats/tree/main/file-formats/sito).

**Comment by huber-nicolas** [comment]
Can you please add an example object for v2?

**Review by huber-nicolas — CHANGES_REQUESTED** [review]
Please see my 2 comments

**Comment by thomas-kaltenbach** [comment]
> Please see my 2 comments

done

**Review by huber-nicolas — APPROVED** [review]

**Comment by huber-nicolas** [comment]
Since this Abap File Format is not yet productively used we can still update the version 1 compatibly.

**Pull request merged**